NitroxServer/Communication/LiteNetLib/LiteNetLibServer.cs

```diff
@@ -47,13 +47,15 @@ public override bool Start()
                 BeginPortForward(portNumber);
             }
 #endif
+            LANDiscoveryServer.Start();
```

Comment:
The `LANDiscoveryServer` should be an feature which can be disabled with `NitroxServer.Serialization.ServerConfig`. In big networks (like school) this feature is maybe not desired.